Sources/Kitura/Router.swift

```diff
@@ -120,7 +120,7 @@ public class Router {
     // MARK: Swagger
     
     /// Contains the structures needed for swagger document generation
-    var swagger: SwaggerDocument
+    public var swagger: SwaggerDocument
```

Comment:
The problem with this approach is that it introduces an ordering problem.  Routes are added to the swagger document each time you call `router.get()` et. al, and so if you were to reassign `router.swagger` after registering some routes, those routes would no longer appear in the document.

Perhaps you could leave this internal, and instead add an additional parameter eg. `apiDocument: SwaggerDocument = SwaggerDocument()` to `router.init`?